### PR TITLE
Update README.md to point to build artifacts

### PR DESCRIPTION
**Pull request by ChucksConsulting**

### CODE REVIEW

**Comment by Ckath** [comment]
probably best to just link to https://github.com/Sneeds-Feed-and-Seed/sneedacity/actions for now since otherwise we have to update the url each time.

if some truly worthwhile things come along we should do an actual release

File: README.md [hunk 1]
```diff
@@ -21,6 +21,9 @@
 - **Accessibility** for VI users.
 - **Analysis and visualization** tools to analyze audio, or other signal data.
 
+## Pre-built binaries
+Binaries for the latest known to work build of Sneedacity are available [here](https://github.com/Sneeds-Feed-and-Seed/sneedacity/actions/runs/1009360152)
```
Comment (Ckath) [inline README.md:25]: probably best to just let it point to https://github.com/Sneeds-Feed-and-Seed/sneedacity/actions for now, so it doesnt go out of date quickly

**Review by Ckath — CHANGES_REQUESTED** [review]

**Comment by ChucksConsulting** [comment]
> probably best to just link to https://github.com/Sneeds-Feed-and-Seed/sneedacity/actions for now since otherwise we have to update the url each time.
> 
> if some truly worthwhile things come along we should do an actual release

Idk, for normal people navigating 1 level of indirection is already asking a lot. It may even be better to just link the files directly on the front page. Plus we don't yet have experimental branches, so this makes it less likely that someone will download a new version that crashes every 5 minutes or something like that.

**Comment by Ckath** [comment]
good enough for now, if we cross some major stuff like wxwidgets and conan we can do a release probably

> > probably best to just link to https://github.com/Sneeds-Feed-and-Seed/sneedacity/actions for now since otherwise we have to update the url each time.
> > if some truly worthwhile things come along we should do an actual release
> 
> Idk, for normal people navigating 1 level of indirection is already asking a lot. It may even be better to just link the files directly on the front page. Plus we don't yet have experimental branches, so this makes it less likely that someone will download a new version that crashes every 5 minutes or something like that.

also fair, guess its good enough for now, not a lot of things are happening anyway. and probably the next worthwhile ones will merrit a proper release

**Pull request merged**